src/server/apps/worldserver/worldserver.conf.dist

```diff
@@ -4467,6 +4468,46 @@ Debug.Arena = 0
 #
 ###################################################################################################
 
+###################################################################################################
+# DYNAMIC RESPAWN SETTINGS
+#
+#
+#    Respawn.DynamicRateCreature
+#        Description: The rate at which the respawn time is adjusted for high player counts in a zone (for creatures).
```

Comment:
First time reading this I thought this was a (1/rate) * respawntime with rate being a float > 1.0f, but the next line mentions it's a player number. Though the config is a float. :confused:  

Reading `ApplyDynamicModeRespawnScaling` clears it up, but the wording can be better here. Worth a rename and/or mentioning the formula?
```
Formula: adjustFactor = rate / playerCount;
```

Reply:
Trinitycore cherry-pick